api/src/test/java/org/pmiops/workbench/api/DataSetControllerTest.java

```diff
@@ -716,27 +728,16 @@ public void testGetPythonQuery() {
                 + "condition_source_concept_id IN (123)) \n"
                 + "AND (c_occurrence.PERSON_ID IN (SELECT * FROM person_id from `"
                 + TEST_CDR_TABLE
-                + ".person` person)) "
+                + ".person` person WHERE "
+                + NAMED_PARAMETER_VALUE.getValue()
```

Comment:
If I'm reading this correctly, this turns into: `WHERE ICD9 IN (2, 5)`. This doesn't seem right

Reply:
In this case, I just threw some random text in. In truth, it is more likely to be something like `WHERE concept_id IN (132543, 2135643...)`

Reply:
Could you change it to be a more realistic value? I think future editors will find this confusing